deepform/data/add_features.py

```diff
@@ -137,7 +146,7 @@ def label_tokens(tokens, labels, max_token_count):
     for col_name, label_value in labels.items():
         tokens[col_name] = 0.0
         match_fn = LABEL_COLS[col_name]
-
+        max_token_count = MAX_TOKENS_BY_TARGET[col_name]
         if col_name == "advertiser":
```

Comment:
Now that you have a max_token_count for each kind, we don't need a conditional -- `label_multitoken` should just be called in all cases, and we can remove the hardcoded check for advertiser.